### PR TITLE
Add URI override as a top-level option

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java [hunk 1]
```diff
@@ -130,6 +133,18 @@ public Builder commonAttributes(Attributes commonAttributes) {
       return this;
     }
 
+    /**
+     * Set a URI to override the default ingest endpoint.
```
Comment (breedx-nr) [inline src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java]: Probably nitpicky, but it feels wrong to call it an "endpoint" and then only use the scheme/host/port portion of the URI.  Maybe it clarifies things to call it a base URI or something similar?

Comment (jkwatson) [reply src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java]: I'm trying to keep the same docs/naming as in the telemetry SDK (from which I copied the javadoc).

File: src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java [hunk 2]
```diff
@@ -81,6 +83,7 @@ public static Builder newBuilder() {
     private SpanBatchSender spanBatchSender;
     private String apiKey;
     private boolean enableAuditLogging = false;
+    private URI uriOverride;
```
[the remark this answers is not in the document]
Comment (jkwatson) [reply src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java]: I was trying as much as possible to make the attribute line up with the method name on the builder, which I wanted to line up with the same thing on the telemetry SDK. Least surprise, and all.

File: src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java (hunk 1 above)
Comment (breedx-nr) [reply src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java]: aight

File: src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (tspring) [reply src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java]: Sorry, I meant to change the method name on the builder as well. If, the Telemetry SDK has a uriOverride method, then I'm fine with this

[the remark this answers is not in the document]
Comment (jkwatson) [reply src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java]: yep, you can see it being called on line 161

**Pull request merged**